src/Pages/HomePage/Elements/Elements.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import img1 from './images/progress.png';
```

Comment:
I would rename these and use more clear names rather than img1 call it `Progress`